src/Components/CountdownTimer/CountdownTimer.js

```diff
@@ -1,21 +1,60 @@
-import React, { useContext } from "react";
-import { SettingsContext } from "../../Context/SettingsContext";
-import { ViewContext } from "../../Context/ViewContext";
-import Timer from "react-compound-timer";
-import style from "./CountdownTimer.module.scss";
-import playTimerIcon from "../../assets/timer/playTimerIcon.png";
-import pauseTimerIcon from "../../assets/timer/pauseTimerIcon.png";
-import skipTimerIcon from "../../assets/timer/skipTimerIcon.png";
-import resetTimerIcon from "../../assets/timer/resetTimerIcon.png";
+import React, { useContext } from 'react'
+import { SettingsContext } from '../../Context/SettingsContext'
+import { ViewContext } from '../../Context/ViewContext'
+import Timer from 'react-compound-timer'
+import style from './CountdownTimer.module.scss'
+
+// Icons ---------->
+import playTimerIcon from '../../assets/timer/playTimerIcon.png'
+import pauseTimerIcon from '../../assets/timer/pauseTimerIcon.png'
+import skipTimerIcon from '../../assets/timer/skipTimerIcon.png'
+import resetTimerIcon from '../../assets/timer/resetTimerIcon.png'
+
+// Sounds ---------->
+import gong from '../../assets/audio/gong.mp3'
```

Comment:
So excited to hear all these! I'm so glad you got to infuse your design & music skills/passions into this project

Reply:
Thank you @Abdeboskey for walking me through the demo of these awesome sound components yesterday. Such an easy BE fix and such a heightened user experience.